chain-cli/network/fablo-config-default.json

```diff
@@ -20,7 +20,8 @@
           "type": "raft",
           "instances": 1
         }
-      ]
+      ],
+      "tools": { "fabloRest": true }
```

Comment:
This is not required for Orderer organization, since technically you are not supposed to call chaincodes from orderer organization